### ISSUE TITLE
Cannot install Microsoft.NETCore.UniversalWindowsPlatform 6.0.7 for MinVersion 10.0.16299.0

### ISSUE THREAD

**Issue by martinsuchan** [Microsoft/dotnet#629]
I have Windows 10 1709, VS 15.5.7.
When I try in my UWP project with both MinVersion/TargetVersion 10.0.16299.0 to update **Microsoft.NETCore.UniversalWindowsPlatform to 6.0.7**, I get this error:
> Unable to find metadata of Microsoft.NETCore.Platforms.2.1.0-preview1-25831-01

I've done some tests:
- installing other versions of this NuGet package works
- if I have other MinVersion than 10.0.16299.0, then I can install NuGet 6.0.7 as well.

It looks like the package 6.0.7 has some non-existent dependencies when targeting Fall Creators Update.

**Comment by lt72** [comment]
Just to confirm, you say that you are using VS 15.5.7.
Did you also used or installed any preview version of VS 15.6?

**Comment by lt72** [comment]
Disregard my question, I see you already confirmed that on Twitter. Thank you for reporting this issue, we will be working on a fix momentarily.

**Comment by zamont** [comment]
Hi Martin, thanks for reporting this. We have a fix in development, but for the time being you can workaround this issue by deleting the following directory on-disk: `C:\Program Files (x86)\Microsoft SDKs\NuGetPackagesFallback\Microsoft.NETCore.UniversalWindowsPlatform\6.0.7` . Please let me know if that does not work.

**Comment by martinsuchan** [comment]
Tried to delete the NuGetPackagesFallback folder, but the bug was still happening. I had to delete both these two locations as well.

> C:\Program Files (x86)\Microsoft SDKs\NuGetPackages\Microsoft.NETCore.UniversalWindowsPlatform\6.0.7\Microsoft.NETCore.UniversalWindowsPlatform.nuspec
> C:\Users\martin.suchan\.nuget\packages\microsoft.netcore.universalwindowsplatform\6.0.7\Microsoft.NETCore.UniversalWindowsPlatform.nuspec

My guess is wrong "poisoned" version of Microsoft.NETCore.UniversalWindowsPlatform 6.0.7 package got installed with VS 15.6 Preview 5 and got copied to several places from the initial preinstalled location. See the attached file for details.
It's interesting to note that we're hitting a problem here that packages installed by Preview version of VS are interfering with stable version of VS. This should not be happening.

[Microsoft.NETCore.UniversalWindowsPlatform.nuspec.txt](https://github.com/Microsoft/dotnet/files/1750879/Microsoft.NETCore.UniversalWindowsPlatform.nuspec.txt)

**Comment by zamont** [comment]
You're right, that is something that should not happen and we need to correct that. We were having a hard time reproducing this error ourselves, so is there any more context you could provide about how and in what order you installed the stable VS and the Preview version? Did you uninstall the Preview version? Did you create your project in the Preview version then re-open it in stable VS? If there are any hints you could provide us here, that would be very helpful.
